diff --git a/src/include/objects.h b/src/include/objects.h
index 81eec29..45b6b58 100644
--- a/src/include/objects.h
+++ b/src/include/objects.h
@@ -554,7 +554,7 @@ struct MotionGlobal_0c2d9260 {
     unsigned char pad7[12 - 7];
     float f12;
     unsigned char pad16[0x88 - 16];
-    float f88, f8c;
+    float f88, f8c, f90;
 };
 
 struct SolHorizontalTarget { unsigned char pad[16]; float x, y; };
