diff --git a/src/include/objects.h b/src/include/objects.h
index 2b9562b..e9a8200 100644
--- a/src/include/objects.h
+++ b/src/include/objects.h
@@ -349,7 +349,9 @@ struct Actor {
     char b4c9;
     unsigned char pad15[0x4dc - 0x4ca];
     unsigned short w4dc;
-    unsigned char pad15b[0x524 - 0x4de];
+    unsigned char pad15b[0x4e0 - 0x4de];
+    unsigned short w4e0;
+    unsigned char pad4e2[0x524 - 0x4e2];
     char b524;
     unsigned char b525;
     unsigned char pad526[0x52c-0x526]; char b52c;
