diff --git a/src/include/objects.h b/src/include/objects.h
index 495059c..f372ac1 100644
--- a/src/include/objects.h
+++ b/src/include/objects.h
@@ -299,8 +299,8 @@ struct Actor {
     unsigned char pad10a[3],b29f;
     unsigned short w2a0;unsigned char pad2a2[2];
     struct ActorSub2a4 sub2a4;
-    unsigned char pad10b[0x2c6 - 0x2a4 - sizeof(struct ActorSub2a4)];
-    short s2c6;
+    unsigned char pad10b[0x2c4 - 0x2a4 - sizeof(struct ActorSub2a4)];
+    int l2c4;
     int l2c8;
     unsigned char pad10c[0x320 - 0x2cc];
     int l320;
@@ -310,14 +310,17 @@ struct Actor {
     unsigned char b328;
     unsigned char pad11a[0x340 - 0x329];
     unsigned short w340;
-    unsigned char pad342[0x348 - 0x342];
+    unsigned short w342;
+    unsigned char pad344[0x348 - 0x344];
     unsigned short w348;
     unsigned short w34a;
     unsigned short w34c;
     unsigned short w34e;
     unsigned short w350;
     unsigned short w352;
-    unsigned char pad354[0x364 - 0x354];
+    unsigned char pad354[0x360 - 0x354];
+    unsigned short w360;
+    unsigned char pad362[2];
     unsigned char x364[8];
     unsigned char x36c[8];
     unsigned char x374[8];
